### PR TITLE
Changed simple benchmark to have multiple transactions in a batch

### PR DESCRIPTION
**Pull request by Nidhiintel**
Signed-off-by: Nidhi Kumari <nidhisolanki44@gmail.com>

### CODE REVIEW

**Comment by Ram-srini** [comment]
@panyu4 @jiangyaoguo @feihujiang Please review these changes

**Comment by Ram-srini** [comment]
@feihujiang @haojun  - Please review this PR

File: benchmark/simple/open.js [hunk 1]
```diff
@@ -9,15 +9,16 @@
...
 module.exports.info  = 'opening accounts';
 
-let accounts = [];
+let account_array = [];
+let txnPerBatch;
 let initMoney;
 let bc, contx;
 module.exports.init = function(blockchain, context, args) {
-    if(!args.hasOwnProperty('money')) {
-        return Promise.reject(new Error('simple.open - "money" is missed in the arguments'));
```
Comment (feihujiang) [inline benchmark/simple/open.js]: Why remove these above lines?

**Comment by feihujiang** [comment]
LGTM but for the two removed lines. @Nidhiintel

**Pull request merged**